--- a/configure	2024-05-26 15:53:08.639431817 +0200
+++ b/configure	2024-05-26 15:54:11.846184695 +0200
@@ -652,7 +652,7 @@
 gl_getopt_required=POSIX
 ac_c_werror_flag=
 gt_needs=
-ac_year2038_required=yes
+ac_year2038_required=no
 enable_year2038=no
 ac_largefile_required=no
 enable_largefile=yes
@@ -8632,7 +8632,7 @@
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $ac_cv_sys_year2038_opts" >&5
 printf "%s\n" "$ac_cv_sys_year2038_opts" >&6; }
 
-ac_have_year2038=yes
+ac_have_year2038=no
 case $ac_cv_sys_year2038_opts in #(
   "none needed") :
      ;; #(
